ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/pderesources.properties

```diff
@@ -488,6 +488,7 @@ SchemaAttributeDetails_use=Use:
 AbstractPluginBlock_counter={0} out of {1} selected
 AbstractTargetPage_setTarget=Set as Active Target Platform
 AbstractTargetPage_reloadTarget=Reload Target Platform
+AbstractTargetPage_openPreferences=Open 'Target Platform' preference (lists all available target)
```

Comment:
I have a feeling that ' needs to be specified as ''.  The wording seems awkward.  No doubt @HannesWell will have a constructive suggestion.

Reply:
You're right, I'll change that to what @laeubi suggested.